CHANGELOG.md

```diff
@@ -12,6 +12,12 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 - `sandstone` to select correct font when font-weight changes in some Indian locales
 
+## [2.0.0-experimental.3] - 2021-10-28
```

Comment:
Don't we need [unrleased] new changelog for this change?

Reply:
Good point! It looks updated when I tested this PR. I will drop this change.